cypari2/string_utils.pyx

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+r"""
+Conversion functions for bytes/unicode
+"""
+
+import sys
+encoding = sys.getfilesystemencoding()
+cdef int PY_MAJOR_VERSION = sys.version_info.major
+
+cpdef bytes to_bytes(s):
```

Comment:
I thought it was wrong to declare an inline function inside a .pyx that is used in other parts of the code (here in auto_instance.pxi)